src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -637,6 +637,18 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     object NumericAttributeOrderingAsc extends Ordering[Attribute] {
       override def compare(x: Attribute, y: Attribute): Int = {
         (x, y) match {
+          //order lists by length
+          case (AttributeValueList(a), AttributeValueList(b)) => a.size.compare(b.size)
+          case (AttributeEntityReferenceList(a), AttributeEntityReferenceList(b)) => a.size.compare(b.size)
+          case (AttributeValueList(a), AttributeEntityReferenceList(b)) => a.size.compare(b.size)
+          case (AttributeEntityReferenceList(a), AttributeValueList(b)) => a.size.compare(b.size)
+
+          //lists are bigger than not-lists
+          case (AttributeValueList(a), _) => 1
+          case (AttributeEntityReferenceList(a), _) => 1
+          case (_, AttributeValueList(b)) => -1
+          case (_, AttributeEntityReferenceList(b)) => -1
+
```

Comment:
I think _ in above block should be AttributeNumber, otherwise AttributeString vs AttributeList will be handled inappropriately